galasa-parent/dev.galasa.framework.api.webui/src/main/java/dev/galasa/framework/api/webui/DynamicStatusStoreSingleton.java

```diff
@@ -0,0 +1,43 @@
+package dev.galasa.framework.api.webui;
+
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+
+import dev.galasa.framework.spi.IDynamicStatusStoreService;
+
+@Component(service = DynamicStatusStoreSingleton.class, immediate = true)
+public class DynamicStatusStoreSingleton {
```

Comment:
I dont't think you need this.    Getting the DSS from the framework is very very quick.   so can do during each servelt call